usb/device/targets/TARGET_NORDIC/TARGET_MCU_NRF52840/USBPhyHw.h

```diff
@@ -0,0 +1,110 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2018-2018 ARM Limited
+ *
```

Comment:
can you please add SPDX identifier to new files

Reply:
year should be 2018-2019

Reply:
Resolved, see 1ce7660